snippets/go/getting_started.go

```diff
@@ -10,15 +10,12 @@ func Start() (*breez_sdk_liquid.BindingLiquidSdk, error) {
 	// ANCHOR: init-sdk
 	mnemonic := "<mnemonic words>"
 
-	// Create the default config
-	config := breez_sdk_liquid.DefaultConfig(breez_sdk_liquid.LiquidNetworkMainnet)
+  // Create the default config, providing your Breez API key
+	config := breez_sdk_liquid.DefaultConfig(breez_sdk_liquid.LiquidNetworkMainnet, "<your-Breez-API-key>")
```

Comment:
I think because the API key is optional, you need to pass it as a reference
```suggestion
    breezApiKey := "<your-Breez-API-key>"
	config := breez_sdk_liquid.DefaultConfig(breez_sdk_liquid.LiquidNetworkMainnet, &breezApiKey)
```